bindings/c/include/conv.hh

```diff
@@ -0,0 +1,46 @@
+#pragma once
```

Comment:
Why the `.hh` extension?

Reply:
No reason other than I thought C++ headers were supposed to have `.hh` extensions.

Reply:
Renamed: https://github.com/elalish/manifold/pull/292/commits/1f77ef77f14fd76ace5d0608a683637fb557c0f2